src/bot/commands.rs

```diff
@@ -217,15 +217,17 @@ pub async fn handle_command(
             let user_ids = whitelist.users();
             let mut users_string = String::with_capacity(user_ids.len());
             for (i, user_id) in user_ids.iter().enumerate() {
+                // apparently won't link properly if user did not interact with the bot:
+                // https://stackoverflow.com/questions/40048452/telegram-bot-how-to-mention-user-by-its-id-not-its-username#comment108737106_46310679
                 users_string.push_str(&format!(
-                    "{}\n",
+                    "{}\\\n",
                     markdown::user_mention(
                         *user_id,
-                        &format!("beeestieee {i} 😎 (the best one!!!)")
```

Comment:
Using something like [`Client::unpack_chat`](https://docs.rs/grammers-client/latest/grammers_client/client/client/struct.Client.html#method.unpack_chat) would be nice, but it requires storing access hash (see [here](https://docs.rs/grammers-client/latest/grammers_client/types/struct.PackedChat.html#structfield.access_hash), it's required for users), which would complicate the whitelist logic a bit... I think it's doable though?? Just having a list of unnamed ppl in the output would become unmanageable pretty fast imo